CMakeLists.txt

```diff
@@ -271,14 +280,15 @@ endif()
 configure_file(${Ginkgo_SOURCE_DIR}/cmake/ginkgo.pc.in
     ${Ginkgo_BINARY_DIR}/ginkgo.pc @ONLY)
 
-# WINDOWS NVCC has " inside the string, add escape charater to avoid config problem.
+# WINDOWS NVCC has " inside the string, add escape character
+# to avoid config problem.
 ginkgo_modify_flags(CMAKE_CUDA_FLAGS)
 ginkgo_modify_flags(CMAKE_CUDA_FLAGS_DEBUG)
 ginkgo_modify_flags(CMAKE_CUDA_FLAGS_RELEASE)
 ginkgo_install()
 
 if(MSVC)
-    # Set path/command with $<CONFIG>
+# Set path / command with $ < CONFIG>
```

Comment:
Is some strange format made by auto-formatting?

Reply:
Thanks for noticing. Yes you are correct it's possible somehow I called clang-format by mistake on that file, which created this weird formatting.